eng/native/configureplatform.cmake

```diff
@@ -86,6 +87,7 @@ if(CLR_CMAKE_HOST_OS STREQUAL Darwin)
     set(CLR_CMAKE_HOST_UNIX_AMD64 1)
     set(CLR_CMAKE_HOST_DARWIN 1)
     set(CMAKE_ASM_COMPILE_OBJECT "${CMAKE_C_COMPILER} <FLAGS> <DEFINES> <INCLUDES> -o <OBJECT> -c <SOURCE>")
+    set(CLR_CMAKE_HOST_OS osx)
```

Comment:
What is the convention for `CLR_CMAKE_HOST_OS` ? This line has it lower cased, and the very next line has it upped cased?

Reply:
Again, this feels pretty unnatural to use the lower-cased shortened OS names in CMake. The convention in CMake is to use Linux, Windows, etc.